app/models/user.rb

```diff
@@ -64,13 +64,27 @@ def is_expert_of?(resource)
     # resource_ids_for_role :resource_expert, resource
     if ling && member_of?(group) && is_expert?(group)
 
-      # is thruthy if either is assigned to that resource or
-      # the resource has nobody set as expert for the moment
-      return group.membership_for(self).has_role?(:expert, ling) || has_no_expert(ling)
+      # is thruthy if is assigned to that resource
+      return group.membership_for(self).has_role?(:expert, ling)
     end
 
   end
 
+  # the user is expert to see some kind of interactions, for example the icon of delete ling
+  def is_expert_to_see?(action, item, can_user_perform_the_action)
```

Comment:
this is the method called in can_see? 
Return true or false if the user can see or not the item. It useful in that part of html that you want to show at user if the user can see it